src/lib/core/CHIPConfig.h

```diff
@@ -1443,6 +1443,16 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
 #define CHIP_CONFIG_MAX_SCENES_CONCURRENT_ITERATORS 2
 #endif
 
+/**
+ * @ def CHIP_CONFIG_MAX_SCENES_TABLE_SIZE
+ *
+ * @brief This defines how many scenes a single endpoint is allowed to allocate in flash memory. This value MUST be greater than 16
+ * per spec and MUST be increased to allow for configuring a greater scene table size from Zap.
+ */
+#ifndef CHIP_CONFIG_SCENES_TABLE_SIZE
+#define CHIP_CONFIG_MAX_SCENES_TABLE_SIZE 32
```

Comment:
Start with 16, let others request more

Reply:
Decreased to 16 by default and increased to 24 for test only.